pkg/sentry/sentry.go

```diff
@@ -71,6 +77,10 @@ func (s *Sentry) Start(ctx context.Context) error {
 		return nil
 	})
 
+	if err := s.startCrons(ctx); err != nil {
```

Comment:
should this be in a goroutine?

Reply:
Nah all good, its handled by the `StartAsync` function in gocron